docs/internals/optimizer.rst

```diff
@@ -1279,8 +1279,8 @@ This is a tiny step that helps in reversing the effects of the SSA transform
 if it is combined with the Common Subexpression Eliminator and the
 Unused Pruner.
 
-The SSA form we generate is detrimental to code generation on the EVM and
-WebAssembly alike because it generates many local variables. It would
+The SSA form we generate is detrimental to code generation on the EVM
```

Comment:
Wouldn't it be better to have here `..to code generation FOR the EVM..`

Reply:
Yeah, but actually, I can just remove the whole EVM bit.